app/src/main/res/layout/activity_signature.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  style="@style/Clinic.TranslucentBackground"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  tools:context="org.simple.clinic.signature.SignatureActivity">
+
+  <RelativeLayout
+    android:id="@+id/relativeLayout"
+    android:layout_width="0dp"
+    android:layout_height="wrap_content"
+    android:layout_margin="@dimen/spacing_16"
+    android:background="@color/white"
+    app:layout_constraintBottom_toBottomOf="parent"
+    app:layout_constraintEnd_toEndOf="parent"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toTopOf="parent"
+    tools:context="org.simple.clinic.signature.SignatureActivity">
+
+    <LinearLayout
+      android:id="@+id/linearLayout"
+      android:layout_width="match_parent"
+      android:layout_height="300dp"
+      android:layout_margin="@dimen/spacing_16"
+      android:layout_marginBottom="@dimen/spacing_24"
+      android:background="@drawable/background_signature"
+      android:orientation="vertical">
+
+      <com.github.gcacace.signaturepad.views.SignaturePad
+        android:id="@+id/drawSignatureFrame"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:penColor="@color/black" />
+
+    </LinearLayout>
+
+    <TextView
+      android:id="@+id/text_sign_header"
+      style="@style/Clinic.V2.TextAppearance.H6.Grey0"
+      android:layout_width="wrap_content"
+      android:layout_height="28dp"
+      android:layout_alignParentTop="true"
+      android:layout_marginStart="@dimen/spacing_40"
+      android:background="@color/white"
+      android:paddingStart="@dimen/spacing_4"
+      android:paddingEnd="@dimen/spacing_4"
+      android:text="@string/screen_signature_sign_in_this_box" />
+
+    <LinearLayout
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_below="@+id/linearLayout"
+      android:layout_marginTop="@dimen/spacing_16"
+      android:layout_marginBottom="@dimen/spacing_16"
+      android:orientation="horizontal">
+
+      <ImageView
+        android:id="@+id/clear_signature"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_weight="1"
+        android:src="@drawable/ic_undo" />
```

Comment:
We should not be using this to set image resource, we have started using `app:srcCompat` since `ImageView` will be inflated as `AppCompatImageView` by AppCompat and we want to avoid any unnecessary issues with the resource.